error_messages.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright 2018-2020 Gamebridge.ai authors. All rights reserved. MIT license.
+
+export const ERROR_MESSAGE_MISSING_PROPERTIES_MAP =
+  "Unable to load serializer properties for the given context";
+
+export const ERROR_MESSAGE_DUPLICATE_PROPERTY_KEY =
+  "This key name is already in use by another property, please use a different name";
+
+export const ERROR_MESSAGE_DUPLICATE_SERIALIZE_KEY =
+  "This serialize key is already in use by another property, please use a different name";
+
+export const ERROR_MESSAGE_SYMBOL_PROPERTY_NAME =
+  "The key name cannot be inferred from a symbol. A value for serializedName must be provided";
```

Comment:
Should we export these are part of `mod.ts`? That way implementors can catch the specific errors if they occur

Reply:
hm, idk do you think it would get used somehow?